frontend/app/src/main.scss

```diff
@@ -0,0 +1 @@
+html { overflow-y: hidden !important; max-height: 100vh; }
```

Comment:
What are we using this for? Is there any risk that overflowing content at the bottom of the page may be hidden (e.g. a lot of things in the "content" area will have padding and margin in the tops which could push them past the "normal" height of the container div).

Reply:
If you check the screenshots above you can clearly see that the scrollbar is in the content area. I am not sure how content in the bottom would disappear if it is in the content area that has a scrollbar?

Vuetify by default would put the scrollbar in `html` meaning that the scrollbar would extend the whole window. In the context of the app, it doesn't make any sense why the scrollbar would cover the `v-app-bar` too.

Reply:
I get the fix and it's good, but I think sometimes there's a catch. I made a fiddle to try to give an example:
https://jsfiddle.net/fq7y4vh3/1/